### PR TITLE
Translated SPARQL interface to DE

### PR DESCRIPTION
**Pull request by hauschke**
**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: https://github.com/vivo-project/VIVO/issues/3990

# What does this pull request do?
Adds labels for German translation of /admin/sparqlquery.

# What's new?
Untranslated labels are now translated, some missing hyphens were added, too.

# How should this be tested?
A German native speaker should have a look. 

# Interested parties
@VIVO-project/vivo-committers, German speaking VIVO users

# Reviewers' expertise
1. Natural language knowledge
    1. German

# Reviewers' report template
**Please update the following template which should be used by reviewers.**
## General comment
A reviewer should provide here comments and suggestions for requested changes if any.
## Testing
A reviewer should briefly describe here how it was tested
## Code reviewing
A reviewer should briefly describe here which part was code reviewed

### CODE REVIEW

**Review by chenejac — DISMISSED** [review]
## General comment
I am not a German speaker, therefore, I only performed syntax and format check.
## Testing
No expertise for this
## Code reviewing
Looks good regarding syntax and format.

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 1]
```diff
@@ -3137,7 +3137,7 @@ uil-data:resource_uri.Vitro
 uil-data:sparql_query_run_query.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
-        rdfs:label       "Run Query"@de-DE ;
+        rdfs:label       "Query starten"@de-DE ;
```
Comment (matthiasluehr) [inline home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:3140]: Could be "Abfrage starten"@de-DE

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 2]
```diff
@@ -3945,7 +3945,7 @@ uil-data:date_time_value_for.Vitro
 uil-data:sparql_query_title.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
-        rdfs:label       "SPARQL Query"@de-DE ;
+        rdfs:label       "SPARQL-Query"@de-DE ;
```
Comment (matthiasluehr) [inline home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:3948]: "SPARQL-Abfrage"@de-DE

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 3]
```diff
@@ -5081,7 +5081,7 @@ uil-data:return_to_the.Vitro
 uil-data:sparql_query.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
-        rdfs:label       "SPARQL Query"@de-DE ;
+        rdfs:label       "SPARQL-Query"@de-DE ;
```
Comment (matthiasluehr) [inline home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:5084]: "SPARQL-Abfrage"@de-DE

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl (hunk 2 above)
Comment (chenejac) [reply home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:3948]:
```suggestion
        rdfs:label       "SPARQL-Abfrage"@de-DE ;
```

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl (hunk 1 above)
Comment (chenejac) [reply home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:3140]:
```suggestion
        rdfs:label       "Abfrage starten"@de-DE ;
```

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl (hunk 3 above)
Comment (chenejac) [reply home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:5084]:
```suggestion
        rdfs:label       "SPARQL-Abfrage"@de-DE ;
```

**Review by chenejac — COMMENTED** [review]
@hauschke I have created code suggestions from @matthiasluehr comments. In the case you decide to adopt some of Matthias comments, you can just click on Commit changes.

**Review by chenejac — APPROVED** [review]

File: home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl
Comment (matthiasluehr) [inline home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl:1]: Looks good. Due to lack of a proper german translation I think we could leave "SPARQL Query Builder" as it is.

**Review by matthiasluehr — APPROVED** [review]
Looks good. Looks good. Due to lack of a proper german translation I think we could leave "SPARQL Query Builder" as it is.

**Pull request merged**